vendor/bat-native-ads/src/bat/ads/internal/privacy/challenge_bypass_ristretto/public_key.h

```diff
@@ -34,7 +34,7 @@ class PublicKey {
   bool operator==(const PublicKey& other) const;
   bool operator!=(const PublicKey& other) const;
 
-  bool has_value() const { return public_key_.has_value(); }
+  bool has_value() const { return public_key_ && public_key_.has_value(); }
```

Comment:
public_key_ and public_key_.has_value() are the same checks

Reply:
Fixed, thanks